commitlint.config.js

```diff
@@ -1 +1,7 @@
-module.exports = {extends: ['@commitlint/config-conventional']}
+module.exports = {
+  extends: ["@commitlint/config-conventional"],
+  ignores: [
+    // dependabot cannot be configured to match commitlint configuration
```

Comment:
Can we include the github issue from the PR description in the comment here for future reference?